docs/modules/ROOT/pages/accounts.adoc

```diff
@@ -198,17 +367,12 @@ $ sncast \
   --calldata 1 2 3
 ```
 
-=== EthAccountComponent
+=== Ethereum Account
 
-:eth-account-mixin: xref:/api/account.adoc#EthAccountComponent-Embeddable-Mixin-Impl[EthAccountMixin]
-:eth-initializer: xref:/api/account.adoc#EthAccountComponent-initializer[initializer]
 :starknetjs: https://www.starknetjs.com/[StarknetJS]
 :starknetjs-commit: https://github.com/starknet-io/starknet.js/commit/d002baea0abc1de3ac6e87a671f3dec3757437b3[d002baea0abc1de3ac6e87a671f3dec3757437b3]
 
-To create an Ethereum-flavored account, implement the {eth-account-mixin} and set up the constructor to initialize the contract.
-Since this is an EthAccount, the {eth-initializer} expects the `EthPublicKey` type (alias for `Secp256k1Point`) to store as the account's public key.
-The contract also requires the `Secp256K1Impl` implementation in order to serialize and deserialize the `EthPublicKey` type.
-A basic Ethereum-flavored account contract looks like this:
+First, let's take the example account we created before and deploy it:
```

Comment:
Since we have the title in the previous line including the Starknet (Ethereum) word, it feels redundant to me, but if you think it reads more natural, I'm happy to update both places.

Reply:
Fair point. Let's keep it as is